src/LuxurySparse.jl

```diff
@@ -1,6 +1,7 @@
 module LuxurySparse
 
 using LinearAlgebra, SparseArrays, Random
+using SparseArrays: AbstractSparseMatrixCSC
```

Comment:
```suggestion
@static if VERSION > v"1.3"
using SparseArrays: AbstractSparseMatrixCSC
end
```

Reply:
Thanks for the fix. This is not compatible with lower Julia versions since we are not going to drop other Julia release any time soon, could you also add the new released `1.4` to travis.yml?